src/ant/README.md

```diff
@@ -62,16 +64,16 @@ about this tool.
 ## Algorithm
 
 | <img src="./doc/images/example_ant.png" width=400px> | <img src="./doc/images/step1.png" width=400px> |
-|:--:|:--:|
-| Antenna Checker Algorithm: WireGraph Example | Step 1: (a) Start from the root node (ITerm) using upper Via to find a node for a new wire. (b) Save the ITerm area for cumulative gate/diffusion area. |
+| --- | --- |
+| <b>Antenna Checker Algorithm</b>: WireGraph Example | <b>Step 1</b>: (a) Start from the root node (ITerm) using upper Via to find a node for a new wire. (b) Save the ITerm area for cumulative gate/diffusion area. |
```

Comment:
Isn't `**` more idiomatic for bold in .md ?  Why are we switching to html markup (<b>)?

Reply:
Changed to `**`.